### PR TITLE
370/ml similar facts endpoint

### PR DESCRIPTION
**Pull request by arekmano**
Added Similar Fact and Outcome values to ML Service Endpoint

### CODE REVIEW

**Review by dodoels — APPROVED** [review]
LGTM

**Review by Vynny — APPROVED** [review]
LGTM

**Review by Samuel-Campbell — APPROVED** [review]
cool cool

File: src/ml_service/web/ml_controller.py [hunk 1]
```diff
@@ -66,13 +68,21 @@ def dict_to_vector(input_dict):
...
-    def vector_to_dict(outcome_vector):
+    def outcome_vector_to_dict(outcome_vector):
         return_dict = {}
         for outcome_index in MlController.classifier_labels:
             label = MlController.classifier_labels[outcome_index][0]
             return_dict[label] = str(outcome_vector[outcome_index])
         return {'outcomes_vector': return_dict}
 
+    @staticmethod
+    def fact_vector_to_dict(fact_vector):
```
Comment (Samuel-Campbell) [inline src/ml_service/web/ml_controller.py:79]: no blocker, should unittest  this. MLController is in our important tests

Comment (arekmano) [reply src/ml_service/web/ml_controller.py:79]: next PR

**Pull request merged**